src/libs/PolicyDistanceRatesUtils.ts

```diff
@@ -0,0 +1,25 @@
+import type {FormInputErrors, FormOnyxValues} from '@components/Form/types';
+import type ONYXKEYS from '@src/ONYXKEYS';
+import * as CurrencyUtils from './CurrencyUtils';
+import getPermittedDecimalSeparator from './getPermittedDecimalSeparator';
+import * as MoneyRequestUtils from './MoneyRequestUtils';
+import * as NumberUtils from './NumberUtils';
+
+type RateValueForm = typeof ONYXKEYS.FORMS.WORKSPACE_RATE_AND_UNIT_FORM | typeof ONYXKEYS.FORMS.POLICY_CREATE_DISTANCE_RATE_FORM;
+
+function validateRateValue(values: FormOnyxValues<RateValueForm>, currency: string, toLocaleDigit: (arg: string) => string): FormInputErrors<RateValueForm> {
```

Comment:
We miss to check the duplicated value and it caused this problem https://github.com/Expensify/App/issues/51769